packages/components/src/Components/TagModal/TagModal.js

```diff
@@ -78,103 +85,59 @@ export default class TagModal extends React.Component {
                             onApply();
                             toggleModal(e, true);
                         }}>
-                            Apply tags
+                            Apply {Array.isArray(tabNames) ? 'selected' : 'tags'}
                         </Button>,
                         <Button key="cancel" variant="link" onClick={(e) => toggleModal(e, false)}>
                             Cancel
                         </Button>
                     ]
                 }}
             >
-                {onUpdateData && <PrimaryToolbar
-                    {...onSelect && pagination && {
-                        bulkSelect: {
-                            count: selected.length,
-                            onSelect: (isSelected) => {
-                                if (isSelected) {
-                                    onSelect(unique([ ...rows, ...selected ]));
-                                } else {
-                                    onSelect(selected.filter(({ id }) => !rows.find(({ id: rowId }) => rowId === id)));
-                                }
-                            },
-                            checked: loaded && calculateChecked(rows, selected),
-                            items: [{
-                                title: 'Select none (0)',
-                                onClick: () => onSelect([])
-                            },
+                {Array.isArray(tabNames) ?
```

Comment:
This can be stored in a variable. There are multiple calls with the same argument.